start_docker.sh

```diff
@@ -68,4 +73,17 @@ run_params=(
 )
 [[ ! -z "$external" ]] && run_params+=(-v "$external:/data/external")
 
-docker run -it "${run_params[@]}" angelolab/ark-analysis:v0.4.2
+# remove container if -e set, need to rebuild container if potential new volumes set
+if [[ ! -z "$external" ]]
+  then
+    # this prevents script from failing in case the container does not already exist
+    docker rm -f $VERSION > /dev/null 2>&1 || true
+fi
+
+# if no Docker container found named $VERSION, create it, otherwise boot it up
+if [[ $(docker ps -a --format "{{.Names}}" | grep $VERSION | wc -l) -eq 0 ]]
+  then
+    docker run -it "${run_params[@]}" --name $VERSION angelolab/ark-analysis:$VERSION
```

Comment:
Will you test all of this logic to make sure it works?
And test it in combination with having made a change using the -u flag previously, to make sure that's carried forward?

Reply:
Updated this logic to recreate the Docker image each time, verified this works on the `-u` flag.